tika-core/src/main/java/org/apache/tika/config/TikaActivator.java

```diff
@@ -40,24 +43,44 @@
 
     private ServiceTracker detectorTracker;
 
+    private ServiceTracker embeddedStreamTranslatorTracker;
```

Comment:
These might not be needed - but - I was checking few of the parser packages and found a package providing an EncodingDetector - so I suggest we keep them to avoid issues with providers not being loaded.